report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,246 @@
+The purpose of the IRF is to calibrate the measured signal to physical units since different pixels have different sensitivities\cite{chemcam_instrument_suite}.
+For example, pixels located at the edge of a CCS usually exhibit lower sensitivity compared to those at the center, and the optics vary in their reflective and absorptive properties at different wavelengths.
+One argument for inverting the IRF is that it introduces more noise in areas of interest on the spectrum by multiplying with a high value as part of the conversion to physical units.
+However, during a conversation with one of the original authors of \citet{cleggRecalibrationMarsScience2017}, criticism was raised against this.
+Specifically, the author pointed out that weighing by the inverse of the discards the alignment between the spectral data collected by the instrument in Los Alamos and the spectral data collected by the instrument on Mars.
+If the same instrument were used for all analyses, one could argue that the IRF (apart from the inverse square law correction) part is irrelevant, but that is not the case here; there are two different instruments --- one in Los Alamos and one on Mars.
+Based on these considerations, we have decided to not weigh by the inverse of the IRF.
+
+In addition, because \citet{cleggRecalibrationMarsScience2017} does not mention how each of the five location datasets are used for each target, we have decided to only use one for each target.
+This likely does not produce as accurate results as using all five location datasets would, since we do not get a full representation of the sample that was shot at by the laser, instead only getting a partial representation from a single location.
+Nevertheless, to avoid deviating significantly from \citet{cleggRecalibrationMarsScience2017}'s methodology or making unfounded assumptions about their data processing, we have opted for this more conservative approach.
+Our goal is to test each of the components of the pipeline individually rather than trying to reproduce the results to absolute perfection.
+
+Similarly, in our replica of the pipeline, we have not incorporated outlier detection in the ICA part of the pipeline primarily because \citet{cleggRecalibrationMarsScience2017} does not describe their implementation sufficiently enough for replication without unsubstantiated assumptions.
+They mention the use of Median Absolute Deviation for this purpose, but the specifics of their approach remain unclear.
+Our decision to omit outlier detection in the ICA phase of our pipeline is also influenced by a preference for retaining a more comprehensive dataset, despite the potential inclusion of outliers.
+Further supporting this decision is input from one of the original authors involved in the ChemCam project.
+This author highlighted the extensive efforts invested in developing the ChemCam calibration dataset, suggesting a low presence of significant outliers.
```

Comment:
This also kind of weakens the PLS-SM outlier removal argument. Perhaps this justifies our idea for removing PLS-SM outlier removal as an experiment?

Reply:
What "PLS-SM outlier removal" argument are you referring to?
Do you wish to make any changes to these sentences or is this just reflection?

I agree though, I think it does justify that idea. No harm in trying to get rid of outlier removal to see what kind of an impact that has IMO.

Reply:
Just reflection